StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/FileHeaderAnalyzers.cs

```diff
@@ -203,16 +203,26 @@ public static void HandleSyntaxTree(SyntaxTreeAnalysisContext context, StyleCopS
                 if (settings.DocumentationRules.XmlHeader)
                 {
                     var fileHeader = FileHeaderHelpers.ParseXmlFileHeader(root);
-                    if (fileHeader.IsMissing)
-                    {
-                        context.ReportDiagnostic(Diagnostic.Create(SA1633DescriptorMissing, fileHeader.GetLocation(context.Tree)));
-                        return;
-                    }
 
-                    if (fileHeader.IsMalformed)
+                    if (fileHeader != null)
                     {
-                        context.ReportDiagnostic(Diagnostic.Create(SA1633DescriptorMalformed, fileHeader.GetLocation(context.Tree)));
-                        return;
+                        // Don't process documentation rules when an <exclude /> tag is present
+                        /*if (fileHeader.GetElement("exclude") != null)
+                        {
+                            return;
+                        }*/
```

Comment:
Why commented? Anyway, you don't need to modify this code. The "file header" is not true XML documentation, and can't have an <exclude/> tag. See https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1633.md